Bixal/bixal-site-drupal
# Feature - BDS-1: Create styleguide

**New component library powered by StorybookJS and twig.** Documents most components with simpler markup, CSS variables, and USWDS tokens.

This work uses USWDS tokens and Twig templates.

Closes #1.

## How to test

- Checkout branch
- Run `npm install` in terminal
- Run `npm start`
- Visit `http://localhost:6008`

You should see a component library with common site components.
